src/pages/studyView/studyPageHeader/studySummary/StudySummary.tsx

```diff
@@ -70,10 +74,37 @@ export default class StudySummary extends React.Component<IStudySummaryProps, {}
         return [];
     }
 
+    @action
+    updateDataDownloadable(content: boolean) {
+        this.dataDownloadable = content;
+    }
+
+    componentDidMount(): void {
+        if (this.props.studies.length === 1) {
```

Comment:
this should probably be in store so that it's properly responsive.   By putting in in componentDidMount we introduce a dependency on the view lifecycle which is unnecessary

Reply:
Can we talk a bit about this more? The component does not have store. Only the list of studies are required by the component. This blob of the code should not belong here. The code should be executed when `this.props.studies` is updated. A reaction for this?